src/Aspire.Hosting.Azure.EventHubs/AzureEventHubsExtensions.cs

```diff
@@ -174,6 +172,63 @@ public static IResourceBuilder<AzureEventHubsResource> RunAsEmulator(this IResou
             configureContainer(surrogateBuilder);
         }
 
+#pragma warning disable ASPIREEVENTING001 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
```

Comment:
Should we just disable this warning globally? It adds a lot of noise, and we are aware that it is an experimental API.

Reply:
Done.